JSON.cpp

```diff
@@ -52,7 +53,7 @@ JSON JSON::parseFromString(std::string String)
 		}
 		String.erase(0, x + 1);
 
-		if ((akt1 == "name") || (akt1 == "hero") || (akt1 == "monsters") || (akt1 == "lore") || (akt1 == "race") || (akt1 == "additional_info")) {
+		if ((akt1 == "name") || (akt1 == "hero") || /*(akt1 == "monsters") ||*/ (akt1 == "lore") || (akt1 == "race") || (akt1 == "additional_info")) {
```

Comment:
Amennyiben a komment nem szükséges, ne maradjon benne. Tanár urak is kódszemétnek definiálták
```suggestion
		if ((akt1 == "name") || (akt1 == "hero") || (akt1 == "lore") || (akt1 == "race") || (akt1 == "additional_info")) {
```

Reply:
Jogos, javítottam! Észre sem vettük hogy bentmaradt...